src/main/java/Duke.java

```diff
@@ -1,10 +1,170 @@
 public class Duke {
+    protected static ArrayList<Task> taskList;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scan = new Scanner(System.in);
+        taskList = new ArrayList<Task>();
+
+        String logo = " ____        _\n"
+                    + "|  _ \\ _   _| | _____\n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("____________________________________________________________\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        System.out.println("____________________________________________________________");
+        String userInput = null;
+        while(!(userInput = scan.nextLine()).equals("bye")) {
+            String[] splitCheck = userInput.split(" ", 2);
+            try {
+                Command inputCommand = validateCommand(splitCheck[0])
+                        ? Command.valueOf(splitCheck[0].toUpperCase())
+                        : Command.INVALID;
+                switch(inputCommand) {
+                case LIST:
+                    listTask();
+                    break;
+                case MARK:
+                    mark(splitCheck);
+                    break;
+                case UNMARK:
+                    unmark(splitCheck);
+                    break;
+                case DELETE:
+                    delete(splitCheck);
+                    break;
+                case TODO:
+                    todo(splitCheck);
+                    break;
+                case DEADLINE:
+                    deadline(splitCheck);
+                    break;
+                case EVENT:
+                    event(splitCheck);
+                    break;
+                default:
+                    throw new DukeException(" ☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                } //end of switch-case
+
+            } catch (DukeException de) {
+                dukeSpeak(de.getMessage());
+            }
+        } // end of while-loop
+
+        dukeSpeak("Bye. Hope to see you again soon!");
+
+    }
+    public static void todo(String[] input) throws DukeException {
+        if (input.length == 1 || input[1].isEmpty()) {
+            throw new DukeException(" ☹ OOPS!!! The description of a todo cannot be empty.");
+        }
+        Todo tempTodo = new Todo(input[1]);
+        taskList.add(tempTodo);
+        String message = "Got it. I've added this task:\n " + tempTodo.toString();
+        message += "\nNow you have " + taskList.size() + " tasks in the list.";
+        dukeSpeak(message);
+    }
```

Comment:
these todo/deadline/event methods maybe you could use more OOP like Ui, Parser,... (according to the website). but it looks good anyways!